tests/src/JIT/Regression/JitBlue/DevDiv_206786/handleMath.il

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Need to be careful about constant folding and handles
+// during prejitting.
```

Comment:
Currently there is a crossgen arg to buildtest.cmd. It will crossgen the tests before running. The default is /ReadyToRun. For this failure to repro the test needs to be crossgen-ed in with /FragileNonVersionable. buildtest.cmd doesn't currently have an arg for that. We should add one.